### PR TITLE
EVG-7777 add drag-and-drop to project commands

### PR DESCRIPTION
**Pull request by ablack12**
image
(Wonky screenshot but in this image "third" is being moved and the bar indicates it's being moved between first and second.)

This works finally!!! Two things:

I'm not sure if the css is more complicated than it needs to be, I was originally going for a triangle but then I messed something up and it gave me this bar and I like it better now.

I want the text boxes to fully utilize the card space and I'm not sure how (I tried using flex/float/md-input/etc and I don't know what I'm doing).

### CODE REVIEW

File: service/templates/projects.html [hunk 1]
```diff
@@ -656,20 +657,25 @@ <h3>Virtual Workstations Commands</h3>
...
-            <div class="col-lg-2">
-              <button class="btn btn-default btn-danger" type="button" ng-click="removeWorkstationCommand($index)">
-                <i class="fa fa-trash"></i>
-              </button>
-            </div>
-          </div>
+          <ol id="workstation-list" class="sortable-list">
+            <li ng-repeat="obj in settingsFormData.workstation_config.setup_commands track by $index">
+              <md-card flex=70>
+                <md-card-content>
```
Comment (john-m-liu) [inline service/templates/projects.html:663]: Could you add a style of cursor:move on this white area? It's not clear that these are draggable

File: public/static/js/projects.js [hunk 2]
```diff
@@ -183,6 +183,31 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
 
   $scope.refreshTrackedProjects($window.allTrackedProjects);
 
+  $(function  () {
+    $("#workstation-list").sortable({
+      group: 'workstation_list',
+      pullPlaceholder: false,
+      // set $item relative to cursor position
+      onDragStart: function ($item, container, _super) {
```
Comment (john-m-liu) [inline public/static/js/projects.js:191]: Not sure if there's a dragEnd event, but something should mark the form as dirty if you move commands around

File: service/templates/projects.html [hunk 3]
```diff
@@ -656,20 +657,25 @@ <h3>Virtual Workstations Commands</h3>
...
-            <div class="col-lg-2">
-              <input class="form-control" ng-model="obj.directory" type="text" placeholder="directory">
-            </div>
-            <div class="col-lg-2">
-              <button class="btn btn-default btn-danger" type="button" ng-click="removeWorkstationCommand($index)">
-                <i class="fa fa-trash"></i>
-              </button>
-            </div>
-          </div>
+          <ol id="workstation-list" class="sortable-list">
```
Comment (john-m-liu) [inline service/templates/projects.html:660]: This list should be bound to something in $scope shouldn't it? I'm seeing that dragging stuff around doesn't save correctly, which I assume is related to this

**Comment by ablack12** [comment]
Closed for reasons detailed in the ticket.

**Pull request closed without merging**